packages/api/server.js

```diff
@@ -28,13 +28,13 @@ const { v4 } = require('uuid');
 
 const cors = require('cors');
 
-global.headCollection = process.env.HEAD_COLLECTION || 'testing/main/repos';
+global.headCollection = process.env.HEAD_COLLECTION || 'production/main/repos';
```

Comment:
I think it's risky to default to `production` data, if it's okay let's stick with 'testing/main/repos'

... we'll be changing this collection any ways I expect 😄

Reply:
I agree, if you need a collection with more data, I think you can use the `staging/main/repos` instead.  That is what @gis727 and I have done